themes/beaver/assets/css/beaver-grid-layout.css

```
@@ -2051,7 +2051,7 @@ ul.pp-advanced-menu-horizontal li.mega-menu > ul.sub-menu ul.sub-menu {
 .fl-node-menu .menu > li.focus > .pp-has-submenu-container > a,
 .fl-node-menu .menu > li.current-menu-item > a,
 .fl-node-menu .menu > li.current-menu-item > .pp-has-submenu-container > a {
-  color: #1a8cff;
+  color: var(--color-primary);
   color: var(--jt-primary, #1a8cff); /* Progressive enhancement with fallback */
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)